docs/building.rst

```diff
@@ -481,6 +481,19 @@ If you enabled test in the configuration step, to run them, run::
 
 	ctest -j8
 
+building with VCPKG
+
+You can download and install libtorrent using the [vcpkg](https://github.com/Microsoft/vcpkg/) dependency manager:
+
+    git clone https://github.com/Microsoft/vcpkg.git
+    cd vcpkg
+    ./bootstrap-vcpkg.sh
+    ./vcpkg integrate install
+    ./vcpkg install libtorrent
```

Comment:
this indented block should work as it is (with the colon I suggest above). But the convention I have in the documentation is to use a single tab character for indentation.

Reply:
Updated.